### PR TITLE
purview admin

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**
apiview https://apiview.dev/Assemblies/Review/d89c9f6e162d4019b17fec981f89e413#

Pending for swagger bug fix https://github.com/Azure/azure-rest-api-specs/pull/16164

samples/README.md is purposely omitted as this can not be released from main branch.

Release will happen on branch https://github.com/Azure/azure-sdk-for-java/tree/feature/purview20211005 (which would include 3 SDKs from PRs)

CI https://dev.azure.com/azure-sdk/public/_build?definitionId=2688
Release https://dev.azure.com/azure-sdk/internal/_build?definitionId=2689

### CODE REVIEW

File: sdk/purview/azure-analytics-purview-administration/README.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+You can authenticate with Azure Active Directory using the [Azure Identity library][azure_identity].
+
+To use the [DefaultAzureCredential][DefaultAzureCredential] provider shown below, or other credential providers provided with the Azure SDK, please include the `azure-identity` package:
+
+[//]: # ({x-version-update-start;com.azure:azure-identity;dependency})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-identity</artifactId>
+    <version>1.3.5</version>
```
Comment (haolingdong-msft) [inline sdk/purview/azure-analytics-purview-administration/README.md:51]: update version?

File: sdk/purview/azure-analytics-purview-administration/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Release History
+
+## 1.0.0-beta.1 (2021-08-23)
```
Comment (haolingdong-msft) [inline sdk/purview/azure-analytics-purview-administration/CHANGELOG.md:3]: ## 1.0.0-beta.1 (Unreleased)

File: sdk/purview/azure-analytics-purview-administration/README.md (hunk 1 above)
Comment (weidongxu-microsoft) [reply sdk/purview/azure-analytics-purview-administration/README.md]: done

**Comment by weidongxu-microsoft** [comment]
beta.1 already release

**Pull request merged**